modules/ROOT/pages/6.6.1-release-notes.adoc

```diff
@@ -122,8 +122,15 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 === On Safari and Firefox browsers, scroll positions were not always maintained when updating the content of a `streamContent: true` iframe dialog component
 //#TINY-10078
 
-// CCFR here.
+On each content update for iframe dialog components with `streamContent: true` set, the iframe is expected to automatically scroll to the bottom if the pre-update scrolled position is already at the end of the scrollable area. It is also expected, however, that the end-user can freely scroll through the iframe as content updates occur.
+
+When Chrome is the host browser, scroll positions are natively maintained after `document.write()`, but this is not the case when either Safari or Firefox are the host browsers. Both these browsers reset the scroll positions to the top.
```

Comment:
```suggestion
On Chrome, scroll positions are natively maintained after updates, but this is not true when on either Safari or Firefox. Both these browsers reset the scroll positions to the top.
```

Reply:
It is important to keep the fact that the content updates are made by `document.write()`, since the scrolling behavior is different when using other methods. @brian-forte Feel free to combine Ben's and my changes to your desire.
```suggestion
When Chrome is the host browser, scroll positions are natively maintained after content updates via `document.write()`, but this is not the case when either Safari or Firefox are the host browsers. Both these browsers reset the scroll positions to the top.
```